ui/admin/app/templates/scopes/scope/policies/policy/index.hbs

```diff
@@ -1,4 +1,39 @@
 {{!
   Copyright (c) HashiCorp, Inc.
   SPDX-License-Identifier: MPL-2.0
-}}
\ No newline at end of file
+}}
+
+<Rose::Layout::Page as |page|>
+  <page.breadcrumbs>
+    <Breadcrumbs::Container />
+  </page.breadcrumbs>
+
+  <page.header>
+    <h2>
+      {{t 'resources.policy.title'}}
+    </h2>
+    <Copyable
```

Comment:
I would suggest to use `Hds::Copy::Snippet` here. If you want, to keep the scope of this PR as it is, perhaps we can add a task to review these [here](https://hashicorp.atlassian.net/browse/ICU-9285)?

Reply:
thanks for the catch. addressed [hrer](https://github.com/hashicorp/boundary-ui/pull/2090)